apps/fabric-website/package.json

```diff
@@ -25,7 +25,7 @@
   ],
   "devDependencies": {
     "@types/es6-promise": "0.0.32",
-    "@types/node": "8.0.26",
+    "@types/node": "^8.10.29",
```

Comment:
Changed for consistency with other packages (so `rush check` runs without errors)

Reply:
BTW if you want to keep the `rush check` state consistent for your repo, you can enable `ensureConsistentVersions = true` in **rush.json**.  We generally recommend this, as it catches a lot of mistakes.

---
In reply to: [236131892](https://github.com/OfficeDev/office-ui-fabric-react/pull/7186#discussion_r236131892) [](ancestors = 236131892)

Reply:
Yeah I think I made that change in rush.json already